_include/analytics.html

```diff
@@ -0,0 +1,11 @@
+<head>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-203084925-1"></script>
```

Comment:
Please confirm the UA and GA tags are connected.

Reply:
Updated the code to include only GA tags